pom.xml

```diff
@@ -416,6 +416,7 @@
           <configuration>
             <excludes>
               <exclude>src/main/resources/META-INF/services/*</exclude>
+              <exclude>src/main/python/AccumuloProxy.egg-info/*</exclude>
```

Comment:
It's fine if these are ignored, but another option might be to add the license header automatically, like the main accumulo repo does.